frontend/src/pages/importVault/ImportVaultView.tsx

```diff
@@ -7,6 +7,8 @@ import { VaultContainer } from '../../gen/vultisig/vault/v1/vault_container_pb';
 import { Vault } from '../../gen/vultisig/vault/v1/vault_pb';
 import { SaveVault } from '../../../wailsjs/go/storage/Store';
```

Comment:
Please don't call the store directly on your view.

Reply:
This PR did not introduce this code.